HousingRepairsOnlineApi/Helpers/RepairQueryHelper.cs

```diff
@@ -8,11 +11,28 @@ namespace HousingRepairsOnlineApi.Helpers
     public class RepairQueryHelper : IRepairQueryHelper
     {
         private readonly ContainerResponse cosmosContainer;
+
         public RepairQueryHelper(ContainerResponse container) => cosmosContainer = container;
 
         public FeedIterator<Repair> GetItemQueryIterator<T>(string repairType, string propertyReference) =>
             cosmosContainer.Container.GetItemQueryIterator<Repair>(GetQueryDefinition(repairType, propertyReference));
 
+        public FeedIterator<Repair> GetRepairSearchIterator(IEnumerable<string> repairTypes, string postcode, string repairId)
+        {
+            var repairTypesUppercase = repairTypes.Select(x => x.ToUpperInvariant());
+
+            var query = cosmosContainer.Container.GetItemLinqQueryable<Repair>().Where(x =>
+                x.Id.ToUpper() == repairId.ToUpper()
+                && x.Postcode.Replace(" ", "").ToUpper() == postcode.Replace(" ", "").ToUpper()
+                && repairTypesUppercase.Contains(x.RepairType.ToUpper())
+                && x.Time.StartDateTime.Date > DateTime.Today.Date
```

Comment:
Does this mean that if an appointment start day is in the past, it won't be returned? 
Not sure if this is desired - just an observation. I can see why you would want this. It's just that date filtering is not mentioned in the ticket.